data-tool/flows/tombstone/tombstone_utils.py

```diff
@@ -642,6 +642,32 @@ def format_users_data(users_data: list) -> list:
     return formatted_users
 
 
+def format_cont_out_data(data: dict) -> dict:
+    cont_data = data.get('cont_out', [])
+    if not cont_data:
+        return {}
+
+    cont_data = cont_data[0]
+    region = cont_data['can_jur_typ_cd']
+
+    if region != 'OT':
+        region = 'FEDERAL' if region == 'FD' else region
+        country = 'CA'
+    else:  # placeholder for other
+        country = 'UNKNOWN'
+        region = ''
+
+    formatted_cont_out = {
```

Comment:
I left this [TODO](https://github.com/bcgov/lear/blob/main/data-tool/flows/tombstone/tombstone_mappings.py#L437) in the code awhile back by referring to what filer does. It seems we need `dissolution_date`.

Reply:
I saw this in filer, not sure that's correct. Let me check if that update is necessary.